tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -10470,6 +10471,7 @@ SpirvEmitter::getSpirvShaderStage(hlsl::ShaderModel::Kind smk) {
   case hlsl::ShaderModel::Kind::Pixel:
     return spv::ExecutionModel::Fragment;
   case hlsl::ShaderModel::Kind::Compute:
+  case hlsl::ShaderModel::Kind::Library:
```

Comment:
Adding a comment to explain why `Library` must be `GLCompute` would be helpful.

Reply:
Good catch. This was leftover from original PR. I've removed it now.